app/src/main/res/values-ru/strings.xml

```diff
@@ -31,13 +31,24 @@
     <string name="auto_cap">"Заглавные автоматически"</string>
     <string name="auto_cap_summary">"Писать первое слово предложения с прописной буквы"</string>
     <string name="select_language">"Язык"</string>
-    <string name="locale_name_en_GB">"английский (Великобритания)"</string>
-    <string name="locale_name_en_US">"английский (США)"</string>
+    <string name="locale_name_en_GB">"Английский (Великобритания)"</string>
+    <string name="locale_name_en_US">"Английский (США)"</string>
     <string name="locale_name_es_US">"Испанский (США)"</string>
     <string name="locale_name_hi_ZZ">"Хинглиш"</string>
     <string name="locale_name_sr_ZZ">"Сербский (латиница)"</string>
-    <string name="subtype_traditional">"классическая"</string>
-    <string name="subtype_compact">"компактная раскладка"</string>
+    <string name="subtype_traditional">традиционный</string>
+    <string name="subtype_compact">"компактная"</string>
+    <string name="subtype_generic_layout"><xliff:g id="LANGUAGE_NAME" example="English (US)">%s</xliff:g> (<xliff:g id="LAYOUT" example="QWERTY">%s</xliff:g>)</string>
```

Comment:
These are the same as in `strings.xml`. Please remove.

Reply:
If I understood correctly, I fixed it.

Reply:
Correct, there's just 4 more below.